client/src/main/java/io/pravega/client/stream/ReaderGroupConfig.java

```diff
@@ -48,10 +42,89 @@ public ReaderGroupConfigBuilder startingTime(long time) {
         *
         * @return Reader group config builder.
         */
-
        public ReaderGroupConfigBuilder disableAutomaticCheckpoints() {
            this.automaticCheckpointIntervalMillis = -1;
            return this;
        }
+
+       /**
+        * Add a stream and its associated start {@link StreamCut} to be read by the readers of a ReaderGroup.
+        * @param scopedStreamName Scoped Name of the Stream.
+        * @param startStreamCut Start {@link StreamCut}
+        * @return Reader group config builder.
+        */
+       public ReaderGroupConfigBuilder stream(final String scopedStreamName, final StreamCut startStreamCut) {
+           if (startingStreamCuts == null) {
+               startingStreamCuts = new HashMap<>();
+           }
+           this.startingStreamCuts.put(Stream.of(scopedStreamName), startStreamCut);
```

Comment:
This PR incorporates #2272 (awesome!) by accepting qualified stream names (`builder.stream("scope/stream")`).   I think we should also allow unqualified stream names (`builder.stream("stream2")`), i.e. use the RG scope by default.

One tiny problem is that the builder doesn't know the RG's scope name at this point, which can be solved with a placeholder value  that is resolved when `createReaderGroup` is called.  Or we could move the RG scope name into this builder.

Reply:
Planning to address this enhancement in a separate PR.

Reply:
Created an issue to track this enhancement. https://github.com/pravega/pravega/issues/2397